src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -209,4 +206,4 @@
     <Reference Include="System.Threading.Tasks" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+</Project>
```

Comment:
The newline character should not be removed.

Reply:
That's VS at work. Is there an issue filed with VS to stop removing the last line? Or at least be consistent?